app/helpers/injection_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module InjectionHelper
   def inject_enterprises
-    inject_json_ams "enterprises", Enterprise.activated.includes(:address).all, Api::EnterpriseSerializer, enterprise_injection_data
+    inject_json_ams "enterprises", Enterprise.activated.includes(address: :state).all, Api::EnterpriseSerializer, enterprise_injection_data
```

Comment:
I think this must have been from when I was using bullet (a gem that flags n+1 queries and unused preloading) while writing my queries and it flagged this as an n+1 where we could preload the association. Sorry it's not really relevant to this feature, should have put it on a different branch.

https://github.com/flyerhzm/bullet

Unfortunately bullet + Spree has some bugs so I stopped using it, maybe it's something to consider if we upgrade Spree though